File: go/behavioral/command/DvdCommandNameStarsOff.go

package command


//DvdCommandNameStarsOff - two of two Concrete Commands

type DvdCommandNameStarsOff struct {
   dvdName *DvdName
}
    
func NewDvdCommandNameStarsOff(dvdNameIn *DvdName) *DvdCommandNameStarsOff {
	return &DvdCommandNameStarsOff{ dvdNameIn }
}    

func (this *DvdCommandNameStarsOff) execute () {
	 this.dvdName.SetNameStarsOff()
} 

File: scala/src/main/scala/Behavioral/Command/DvdCommandNameStarsOff.scala
package Behavioral.Command;


//DvdCommandNameStarsOff - two of two Concrete Commands

public class DvdCommandNameStarsOff extends CommandAbstract {  
   private DvdName dvdName;  
    
   public DvdCommandNameStarsOff(DvdName dvdNameIn) {
       this.dvdName = dvdNameIn;
   }    
   public void execute() {
       this.dvdName.setNameStarsOff();
   } 
}